docs/task-parameters.md

```diff
@@ -0,0 +1,49 @@
+## Task Parameters
```

Comment:
amaaaaazing

next time we touch this we should add a link from the main README to here, otherwise this would be hard to find